pandas/tests/io/json/test_pandas.py

```diff
@@ -1012,77 +1012,86 @@ def test_convert_dates_infer(self):
             result = read_json(dumps(data))[["id", infer_word]]
             assert_frame_equal(result, expected)
 
-    def test_date_format_frame(self):
+    @pytest.mark.parametrize(
```

Comment:
Slightly orthogonal as this entire module could use some parametrization but added these here as they were helpful in debugging and I think easy to grok diff

Reply:
Given the difficulty of this PR, I'd say anything that can be broken off should be.